docs/src/explanations/printing.md

```diff
@@ -15,16 +15,25 @@ Chisel provides the `printf` function for debugging purposes. It comes in two fl
 
 Chisel also supports printf in a style similar to [Scala's String Interpolation](http://docs.scala-lang.org/overviews/core/string-interpolation.html). Chisel provides a custom string interpolator `cf` which follows C-style format specifiers (see section [C-style](#c-style) below).
 
-Note that the Scala s-interpolator is not supported in Chisel constructs and will throw an error:
+Note that the Scala s-interpolator is not supported in Chisel.
+Future versions of Chisel will throw an error when using the s-interpolator
+within a printf.
 
 ```scala mdoc:invisible
 import chisel3._
+
+//TODO: Make the following Module fail when using s-interpolator in a printf
+// This used to fail under 2.12 but the macro that implemented this does
+// not work properly in Scala 2.13
+// When fixed add mdoc:fail to following ```scala block and clean up
```

Comment:
This TODO is going to show up in the output mdoc, is that intentional?

Reply:
@chick pointed out that this is in an invisible block so it does not actually show up.